### ISSUE TITLE
new window on click

### ISSUE THREAD

**Issue by tgg-manish** [lxn/walk#363]
Hi guys 
can anyone tell me how can I change MainWindow layout when clicking on the pushbutton?

**Comment by lxn** [comment]
I'm not sure I understand the question. Maybe the multiplepages example could be of help?

**Comment by tgg-manish** [comment]
yes, I am trying for multiple pages. can you please give me another example
or link.
